app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,10 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
-    </div>
+      <% if quote.created_at %>
+        <%= time_ago_in_words(quote.created_at) %> ago
+      <% else %>
+        ago
+      <% end %>
```

Comment:
Under what conditions should a quote not have a `created_at` property? This feature won't be complete until we structure our app such that `completed_at` is __always__ not `nil`. If the seed data contains this error please update the `seeds.rb` file with your fix